### ISSUE TITLE
MyPaint crashes on launch

### ISSUE THREAD

**Issue by ryandesign** [mypaint/mypaint#114]
MyPaint 1.1.1.alpha.20140929+git.4fafd2ad crashes on launch. Note also that glib 2.42.0 is installed. Here is the beginning of the crash log:

```
Process:               Python [90569]
Path:                  /Applications/MacPorts/MyPaint.app/Contents/MacOS/MyPaint
Identifier:            com.github.mypaint.mypaint.MyPaint
Version:               ???
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           Python [90569]
User ID:               502

Date/Time:             2014-10-11 07:17:52.716 -0500
OS Version:            Mac OS X 10.10 (14A386b)
Report Version:        11
Anonymous UUID:        A7ACF2CC-76F2-8D83-0F47-D1D925232A18

Sleep/Wake UUID:       D57CAB1B-6CAE-4ECD-B1A1-D04F9FD6E6C9

Time Awake Since Boot: 140000 seconds
Time Since Wake:       7700 seconds

Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_BAD_ACCESS (SIGSEGV)
Exception Codes:       KERN_INVALID_ADDRESS at 0x0000000000000000

VM Regions Near 0:
--> 
    __TEXT                 0000000107d2b000-0000000107d2d000 [    8K] r-x/rwx SM=COW  /opt/local/Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python

Thread 0 Crashed:: Dispatch queue: com.apple.main-thread
0   ???                           	000000000000000000 0 + 0
1   libglib-2.0.0.dylib           	0x0000000109019ff8 g_hash_table_lookup + 39
2   libgirepository-1.0.1.dylib   	0x0000000109fa34e9 enumerate_namespace_versions + 309
3   libgirepository-1.0.1.dylib   	0x0000000109fa3288 g_irepository_enumerate_versions + 43
4   _gi.so                        	0x0000000109f61ac2 _wrap_g_irepository_enumerate_versions + 71
5   org.python.python             	0x0000000107ddc021 PyEval_EvalFrameEx + 19649
6   org.python.python             	0x0000000107dd7052 PyEval_EvalCodeEx + 1538
7   org.python.python             	0x0000000107d62b6c function_call + 364
8   org.python.python             	0x0000000107d3d033 PyObject_Call + 99
9   org.python.python             	0x0000000107d4a226 instancemethod_call + 182
10  org.python.python             	0x0000000107d3d033 PyObject_Call + 99
11  org.python.python             	0x0000000107d3d4be PyObject_CallMethod + 318
12  org.python.python             	0x0000000107df2bb0 find_module + 336
13  org.python.python             	0x0000000107df6598 import_submodule + 216
14  org.python.python             	0x0000000107df63d7 ensure_fromlist + 583
15  org.python.python             	0x0000000107df3ee6 PyImport_ImportModuleLevel + 1382
16  org.python.python             	0x0000000107dd2227 builtin___import__ + 135
17  org.python.python             	0x0000000107d3d033 PyObject_Call + 99
18  org.python.python             	0x0000000107dd96b9 PyEval_EvalFrameEx + 9049
19  org.python.python             	0x0000000107dd7052 PyEval_EvalCodeEx + 1538
20  org.python.python             	0x0000000107dd6a46 PyEval_EvalCode + 54
21  org.python.python             	0x0000000107df27a1 PyImport_ExecCodeModuleEx + 241
22  org.python.python             	0x0000000107df597f load_source_module + 1055
23  org.python.python             	0x0000000107df65d2 import_submodule + 274
24  org.python.python             	0x0000000107df63d7 ensure_fromlist + 583
25  org.python.python             	0x0000000107df3ee6 PyImport_ImportModuleLevel + 1382
26  org.python.python             	0x0000000107dd2227 builtin___import__ + 135
27  org.python.python             	0x0000000107d3d033 PyObject_Call + 99
28  org.python.python             	0x0000000107dd96b9 PyEval_EvalFrameEx + 9049
29  org.python.python             	0x0000000107ddeb72 fast_function + 274
30  org.python.python             	0x0000000107ddac1d PyEval_EvalFrameEx + 14525
31  org.python.python             	0x0000000107dd7052 PyEval_EvalCodeEx + 1538
32  org.python.python             	0x0000000107dd6a46 PyEval_EvalCode + 54
33  org.python.python             	0x0000000107e00884 PyRun_FileExFlags + 164
34  org.python.python             	0x0000000107e00401 PyRun_SimpleFileExFlags + 769
35  org.python.python             	0x0000000107e15e98 Py_Main + 3096
36  libdyld.dylib                 	0x00007fff849a05c9 start + 1
```

**Comment by achadwick** [comment]
Hi @ryandesign. Can I assume this is fixed now that mypaint/libmypaint#13 is merged?

**Comment by achadwick** [comment]
**Triage note:** No update for nearly 4 calendar months; believed fixed. Reopen and shout at us if it can be reproduced with current master, please.
